README.md

```diff
@@ -74,7 +74,7 @@ Setting up a cobalt deployment comprises of 5 general steps.
 1. You can follow these [instructions](devops/providers/azure-devops/README.md) to create an cloud-based CI pipeline definition.
 2. Our cloud deployment templates provide an configurable default setup intended for a t-shirt sized environment scenario. Pick the template folder most relevant to your use-case as a starting point. Each template folder is shipped with setup instructions.
 3. It's important to implement quality assurance that validates E2E functional assertions against your infrastructure resources. Each template comes pre-packaged with some basic integration and unit tests. We encourage you to define integration tests in the `test/integration` folder of your template that's specific to your use-case.
-4. Follow these [instructions](test-harness/README.md) to setup your local environment. Make sure that the repository lives in the following directory: `$GOPATH/src/github.com/microsoft/cobalt/`
+4. Follow these [instructions](test-harness/README.md) to setup your local environment. Make sure that the repository lives in a directory that does not live within `$GOPATH`.
```

Comment:
Is the project being relocated due to the 'mod' dependency?

Reply:
Indeed it is. You cannot create a go module from within your `$GOPATH`

```bash
$ go mod init foo
go: modules disabled inside GOPATH/src by GO111MODULE=auto; see 'go help modules'
```

This is actually awesome because prior to this you *had* to have your code checked out into a very specific path based on the prior package manager, `dep`. The go community is moving towards `go mod` to avoid this and other related issues.

To update your project, you just need to pull these changes and move your repository outside of your go-path